cmd/nomos/status/status.go

```diff
@@ -148,9 +148,10 @@ func clusterStates(ctx context.Context, clientMap map[string]*ClusterClient) (ma
 			stateMap[name] = unavailableCluster(name)
```

Comment:
This function name is identical to the one in client.go, which could cause confusion. Let's refactor it to be more specific about its purpose here.

Reply:
Ah I misread, it's clusterStates and clusterStatus, nevermind.